src/CMakeLists.txt

```diff
@@ -489,6 +489,12 @@ if(BUILD_TESTING)
   osh_add_exe(coarsen_test)
   test_func(run_coarsen_test 1 ./coarsen_test)
 
+  if(Omega_h_USE_Gmsh)
```

Comment:
Until (if?) we make Kokkos a required dependency, we'll have to add `Omega_h_USE_Kokkos` to this conditional as `Kokkos::parallel_reduce` is used in the test code.

Note, I attempted, and failed, to restore (I don't think the function was ever used...) the `Omega_h::parallel_reduce(...)` wrapper in this branch: https://github.com/SCOREC/omega_h/tree/cws/reduce.

Reply:
I am sorry I forgot to add this flag. Thank you.